topostats/plotting_dictionary.yaml

```diff
@@ -192,61 +192,62 @@ grain_mask_image:
   image_type: "non-binary"
   savefig_dpi: 100
   core_set: false
-# DNA Tracing Troubleshooting Images
-orig_grains:
+# Disordered Tracing Troubleshooting Images
+orig_grain:
   filename: "20-original_grains"
   title: "Image with Threshold Mask"
   image_type: "non-binary"
-  mask_cmap: "green_black"
+  mask_cmap: "blu"
   core_set: false
-smoothed_grains:
+smoothed_grain:
   filename: "21-smoothed_grains"
   title: "Image with Gaussian-Smoothed Threshold Mask"
   image_type: "non-binary"
-  mask_cmap: "green_black"
+  mask_cmap: "blu"
   core_set: false
-orig_skeletons:
+skeleton:
   filename: "22-original_skeletons"
   title: "Image with Overlapping Original Skeletons"
   image_type: "non-binary"
-  mask_cmap: "green_black"
+  mask_cmap: "blu"
   core_set: false
   savefig_dpi: 600
-pruned_skeletons:
+pruned_skeleton:
   filename: "23-pruned_skeletons"
   title: "Image with Overlapping Pruned Skeletons"
   image_type: "non-binary"
-  mask_cmap: "green_black"
+  mask_cmap: "blu"
   core_set: false
   savefig_dpi: 600
+# Nodestats troubleshooting images
 nodes:
   filename: "24-nodes"
   title: "Image with Skeletons and Highlighted Nodes"
   image_type: "non-binary"
-  mask_cmap: "green_black"
+  mask_cmap: "blu_purp_green"
```

Comment:
I don't think its any great burden to avoid abbreviations and be explicit and suggest using `blue_purple_green`.

IDE's all support Tab-completion and its simple to refactor across a whole project with the language servers that sit behind them. If you use VSCode its [F2](https://code.visualstudio.com/docs/editor/refactoring#_rename-symbol).

Reply:
done!